data/com.endlessm.Metrics.xml

```diff
@@ -151,5 +151,48 @@ License along with eos-metrics.  If not, see
       the tracking ID can only be done by a privileged process.
     -->
     <method name="ResetTrackingId"/>
+
+    <!--
+      StartAggregateTimer:
+      @event_id: event type UUID, as an array of 16 bytes
+      @aggregate_key: a key to aggregate events
+      @has_payload: whether the start event has a payload
+      @payload: optional payload for the start event
+      @timer_object_path: object path to an aggregate timer
+
+      Starts an event aggregation timer.
+
+      @payload is ignored if @has_payload is FALSE. (This is to compensate for
+      DBus's lack of maybe types.)
+
+      @aggregate_key is the domain-specific key to aggregate. It can be, for
+      example, the application id when tracking applications, or the unix user
+      id when tracking session times.
```

Comment:
I wonder whether we can do this slightly differently – always aggregate by `payload` but only send it to the server if `has_payload` is true? I think in practice, payload will be the same as aggregate_key if has_payload is true.